Kernel/Devices/Storage/AHCI/Definitions.h

```diff
@@ -371,7 +371,7 @@ class PortInterruptEnableBitField {
     u32 volatile& m_bitfield;
 };
 
-struct [[gnu::packed]] PortRegisters {
+struct PortRegisters {
```

Comment:
We can probably remove the `[[gnu::packed]]`s from other structs in this file as well. 
As `[[gnu::packed]]` can cause worse codegen in general on some architectures, since bytewise loads are slower.
The `AssertSize`s should catch any unwanted padding bytes.